stl/inc/string

```diff
@@ -24,7 +24,7 @@ _STL_DISABLE_CLANG_WARNINGS
 _STD_BEGIN
 // FUNCTION TEMPLATE getline
 template <class _Elem, class _Traits, class _Alloc>
-basic_istream<_Elem, _Traits>& getline(basic_istream<_Elem, _Traits>&& _Istr,
+_NODISCARD basic_istream<_Elem, _Traits>& getline(basic_istream<_Elem, _Traits>&& _Istr,

```

Comment:
`getline` should definitely **not** be marked as `nodiscard`. It's perfectly OK to discard it's return value since it's real result is passed in `_Str`. Ditto 3 overloads below.

Reply:
Oops ,  apologies for the same . I thought it would be necessary so am removing them as you said. Pls do confirm if rest are fine???